sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -1332,9 +1337,13 @@ object SQLContext {
    * Create a SQLListener then add it into SparkContext, and create an SQLTab if there is SparkUI.
```

Comment:
If this method is going to be a no-op under certain circumstances then those circumstances should be documented here.